public/js/evaluations.js

```diff
@@ -437,16 +437,67 @@ function displayDopeEvals(data) {
   $("#displayDopeEvals").append(appendStr);
 }
 
+
 function displayDopeComments(data) {
   if (data.length == 0) return;
-  var appendStr = "<ul>";
+  var appendStr = ""
+
+  // display all comments in a list
+  appendStr += "<ul>";
+  for (i = 0; i < data.length; i++) {
+    if (data[i].comments && data[i].comments.length != 0) {
+      appendStr += "<li class='dopeComment' tabindex='0'>";
+      appendStr += data[i].comments + "</li>";
+    }
+  }
+  appendStr += "</ul>"
+
+  // display details of each submission in a collapsible
+  // TODO: we might also want to have a field for course and professor, for profs who teach more than one course or vice versa.
```

Comment:
I think this new code for assembling+showing the individual reviews should go into its own function, maybe something like `displayIndividualReviews`. That will keep the functions for the different parts of the display separate. 

You'll also need to create a new div on the evaluations page that holds this, and then use that in a line that corresponds to `$("#displayDopeComments").append(appendStr + "</ul></div>");`

Reply:
Done in next revision!